lib/mesh.js

```
@@ -77,13 +78,21 @@ var net    = require('net'),
   }
 
   /*
+   * Special event handler for errors
+   */
+  mesh.error = function(callback){
+    error_handler = callback;
+  }
+
+  /*
    * An error occured in the TCP client
    */
   var erred = function(e){
     if (e.code == 'ECONNREFUSED') {
       console.log('Connection refused to mesh server.');
-    } else {
-      console.log(e);
+    }
```

So if !error_handler and e.code != 'ECONNREFUSED' the error is suppressed?